okhttp/src/jvmTest/java/okhttp3/internal/ws/WebSocketHttpTest.java

```diff
@@ -22,6 +22,7 @@
 import java.net.ProtocolException;
 import java.net.SocketTimeoutException;
 import java.time.Duration;
+import java.util.List;
```

Comment:
accidental import?
just happened to glance at this PR. pardon the unsolicited comment.

Reply:
No it's for 

```
-        .protocols(asList(Protocol.HTTP_1_1))
+        .protocols(List.of(Protocol.HTTP_1_1))
```

Reply:
whoops, i read that backwards, sorry. of course.